armstrongsouljah/storemanager-frontend
# chore(testing): Setup unit testing

#### What does this Pr do?
-Introduces an implementation to enable unit testing.

#### Description of the task to be implemented.
- Install testing libraries
- Add configuration files for testing libraries
- Write initial test

#### How can this be manually tested?
- Fetch and check into the branch locally by running `git fetch origin ch-unit-testing-164352872 && git checkout ch-unit-testing-164352872`
- Install project dependencies by running `yarn install`
- Run the tests by running  `yarn test:coverage`

#### What are the related pivotal tracker stories?
[#164352872](https://www.pivotaltracker.com/story/show/164352872)
